themes/digital.gov/layouts/communities/list.html

```diff
@@ -23,33 +23,98 @@ <h1>{{- .Title | markdownify -}} </h1>
   </div>
 
 
-  <section class="communities-list usa-section">
+  <section class="usa-section">
     <div class="grid-container-desktop">
       <div class="grid-row tablet-lg:grid-gap-2">
+          <div class="grid-col-12">
+            <ul class="usa-card-group">
+              {{/* Gets all communities pages */}}
+              {{ $dg_communities := (where .Site.Pages "Section" "communities").ByTitle }}
+
+              {{/* Gather only the pages with a weight greater than or equal to 1 */}}
+              {{ $dg_communities := (where $dg_communities ".Params.weight" "ge" 1) }}
+
+              {{/* Gather only the pages with a dg-emphasis key */}}
+              {{ $dg_communities := (where $dg_communities ".Params.dg_highlight" "eq" true) }}
+
+              {{ range $i, $e := $dg_communities }}
+
+              <li class="usa-card tablet:grid-col-4">
+                <div class="usa-card__container">
+                  <div class="usa-card__header">
+                    <h2 class="usa-card__heading">{{- $e.Title | markdownify -}}</h2>
+                  </div>
+                  <div class="usa-card__media">
+                    <div class="usa-card__img">
+                      <img
+                        src="{{- $e.Params.dg_image | absURL -}}"
+                        alt="{{- $e.Title }}"
```

Comment:
Let's use a more specific alt description for the actual images themselves instead of the title (it would read twice).

Alternatively, we could use an empty alt and add `aria-hidden="true"` for this decorative image.